recipe/bld.bat

```diff
@@ -221,14 +221,13 @@ if "%PKG_NAME%" == "libtorch" (
         if %ERRORLEVEL% neq 0 exit 1
     )
 
-    @REM Delete all files from the lib directory that do not start with torch_python
-    for %%f in (%SP_DIR%\torch\lib\*) do (
-        set "FILENAME=%%~nf"
-        if "!FILENAME:~0,12!" neq "torch_python" (
-            del %%f
-            if %ERRORLEVEL% neq 0 exit 1
-        )
-    )
+    @REM Move libtorch_python and remove the lib directory afterwards.
```

Comment:
The python-specific lib doesn't have to go into `%LIBRARY_BIN%` necessarily. We should match what the unix build does there.

Reply:
Are we talking of `.lib` or both `.lib` and `.dll`? I think only the former would be needed for linking, and that one should indeed be tiny.

Reply:
`.lib` is 220k, `.dll` is 14M.

Reply:
That sounds good to me, but it probably doesn't work with a simple copy then. I haven't looked into how to change the path that the `.lib` expects the `.dll` in, but I assume it's possible

Reply:
But does it contain a path at all? At least according to a quick look, it seems to contain just the `torch_python.dll` filename, and I can't grep any `site-packages` or `python3` in it either.

Reply:
OK, fair enough. Let's try just copying the `.lib`